--- 1/Makefile
+++ 2/Makefile
@@ -74,7 +74,7 @@
 
 SRC_DIR=src
 
-COREUTILS = basename cat chmod cksum csplit cut echo expand factor false fmt fold head id join kill link ln logname mkfifo mkdir mknod nl od paste pathchk printf ptx pwd readlink rmdir seq sleep split sum tail tee tr tsort unexpand unlink wc whoami test touch true yes
+COREUTILS = basename cat chmod cksum csplit cut echo expand expr factor false fmt fold head id join kill link ln logname mkfifo mkdir mknod nl od paste pathchk printf ptx pwd readlink rmdir seq sleep split sum tail tee tr tsort unexpand unlink wc whoami test touch true yes
 
 BINARIES = $(addprefix $(SRC_DIR)/, $(COREUTILS))
 
--- 1/src/expr.c
+++ 2/src/expr.c
@@ -44,6 +44,8 @@
 
 #define AUTHORS "Mike Parker"
 
+#define strcoll strcmp
+
 #undef NEW
 #define NEW(Type) XMALLOC (Type, 1)
 #define OLD(x) free ((char *) x)
